client/state/jetpack-connect/test/reducer.js

```diff
@@ -29,6 +29,28 @@ import reducer, {
 	jetpackConnectSessions
 } from '../reducer';
 
+const successfulSSOValidation = {
+	type: JETPACK_CONNECT_SSO_VALIDATION_SUCCESS,
+	success: true,
+	blogDetails: {
+		domain: 'website.com',
+		title: 'My BBQ Site',
+		icon: {
+			img: '',
+			ico: '',
+		},
+		URL: 'https://website.com',
+		is_vip: false,
+		admin_url: 'https://website.com/wp-admin'
+	}
+};
+
+const falseSSOValidation = {
+	type: JETPACK_CONNECT_SSO_VALIDATION_SUCCESS,
```

Comment:
In this case, the request was successful and is returning a boolean value. So, I'm not sure that `JETPACK_CONNECT_SSO_VALIDATION_FAILED` fits either.